components/LandingItems.tsx

```diff
@@ -0,0 +1,138 @@
+import { dummyData, Event } from "@/texts/common/dummy"
+import Image from "next/image"
+import { ChangeEvent, createContext, useContext, useEffect, useState } from "react"
+import styled from "styled-components"
```

Comment:
```suggestion
import Image from "next/image"
import { ChangeEvent, createContext, useContext, useEffect, useState } from "react"
import styled from "styled-components"
import { dummyData, Event } from "@/texts/common/dummy"
```
rearrange import sequence in accordance to the [style guide](https://www.notion.so/heads-of-rhdevs/RHDevs-Frontend-Code-Style-Guide-b5f294c047f14f78b370031afb2c73c5)

Reply:
my prettier sets the order though haha